core/domain/exp_domain.py

```diff
@@ -1321,13 +1309,8 @@ def get_training_data(self):
         training_data = []
```

Comment:
I think you need to rename this, since it's conceptually different from the training_data referenced on line 1313.

Reply:
Done